app/middleware/locals.js

```diff
@@ -5,6 +5,7 @@ module.exports = config =>
     res.locals.GOOGLE_ANALYTICS_TRACKING_ID = config.googleAnalyticsId;
     res.locals.WEBTRENDS_ANALYTICS_TRACKING_ID = config.webtrendsId;
     res.locals.HOTJAR_ANALYTICS_TRACKING_ID = config.hotjarId;
+    res.locals.RESULTS_LIMIT = config.resultsLimit;
```

Comment:
Should `RESULTS_LIMIT` be lower case, along with the `***_ANALYTICS_TRACKING_ID` properties?

Reply:
Seems that things that come from ENV files are uppercase, so maintaining that convention.

Reply:
What do you mean in the ENV files?
I thought it had been agreed to remove all upper casing of variables within the application.